README.md

```diff
@@ -30,8 +30,9 @@ You can start the application with:
 `./startup.sh`
 
 ### Running the tests
+Test are run in docker environment pegged to use firefox-esr and can be run as follows:
 
-`./pre-commit.sh` will run the tests.
+`./pre-commit-local.sh` will run the tests.
```

Comment:
FWIW if doing this in docker works fine, then it's better to just have one way of doing things, so I'd say replace pre-commit.sh with the new stuff. You don't want them out of sync.

Also the naming is a bit confusing, because to me, "local" suggests the opposite of docker, i.e. I'm either running it on my local machine, or on a docker machine. That could just be me :-)

Reply:
It's because of this, the container on concourse for self-service would most likely not have the ability to run docker at the moment. The naming, I agree is not perfect but since I opted for this with verify-frontend I decided to keep it this inline.
https://gds.slack.com/archives/CAFAEK63Y/p1599139419160600